css/styles.css

```diff
@@ -0,0 +1,127 @@
+
+.containertwo {
+  background-color: #562B81;
+  height: 50px;
+  padding: 1px;
+  display: flex;
+}
+.heading-text {
+  margin: auto;
+  flex-basis: 43%;
+  color: white;
+  font-size: 20px;
+  font-family: Proza Libre;
+  font-weight: 100;
+  background-color: #562B81;
+}
+
+p {
+  font-family: sans-serif;
+}
+
+.new-entry {
+  font-family: sans-serif;
+  color: rgb(91, 89, 89);
+  font-size: 23px;
+}
+
+img:nth-child(1) {
+  height: 300px;
+  width: 300px;
+}
+
+.titletext {
+  height: 2rem;
+  border-radius: .2rem;
+  border: solid 1px rgb(174, 173, 173);
+  margin-right: 2rem;
+}
+.title-text:focus, .photo-text:focus,
+.textarea-1:focus {
+  outline: none;
+  border: 2px solid #562B81;
+}
+
+
+.phototext {
+  height: 2rem;
+  border-radius: .2rem;
+  border: solid 1px rgb(174, 173, 173);
+  margin-right: 2rem;
+}
+
+.notestext {
+  height: 5rem;
+  border-radius: .2rem;
+  border: solid 1px rgb(174, 173, 173);
+  margin-right: 2rem;
+  padding-bottom: 3.2rem;
+  padding-left: .5rem;
+}
+
+.column-half {
+  flex-basis: calc(20% / 3);
+}
+
+.textarea1 {
+  resize: none;
+  height: 9rem;
+  border-radius: .2rem;
+  border: solid 1px rgb(174, 173, 173);
+  margin-right: 2rem;
+  padding-bottom: 3.2rem;
+  padding-left: .5rem;
+  padding-top: 5px;
+}
+
+  .save-button {
+    width: 52px;
+    height: 30px;
+    padding: 5px;
+    margin-top: 1rem;
+    margin-left: 35.3rem;
+    color: white;
+    background-color: #562B81;
+    border: solid 1px;
+    border-radius: .3rem;
+  }
+
+
+
+
+
+
+
+
+@media screen and (max-width: 650px) {
+  .heading {
+    width: 100%;
+    height: 70px;
+  }
+
+  .heading-text {
+    margin-left: 1.4rem;
+    font-size: 30px;
+  }
+
+  .textarea1 {
+    width: 20.5rem;
+  }
+
+  .heading-text {
+    flex-basis: auto;
+  }
+
+
+  input {
+    width: 20.5rem;
+  }
+  img:nth-child(1) {
+    height: 329px;
```

Comment:
it is best to set only a width on an image, and let the height set itself automatically to prevent aspect ratio issues.